alexandria/data_structures/src/data_structures.cairo

```diff
@@ -6,8 +6,10 @@ use array::ArrayTrait;
 /// * `end` - The index to end the slice at (not included).
 /// # Returns
 /// * `Array<u256>` - The slice of the array.
-fn array_slice(src: @Array<u256>, mut begin: usize, end: usize) -> Array<u256> {
-    let mut slice = ArrayTrait::new();
+fn array_slice<T, impl TDrop: Drop<T>, impl TCopy: Copy<T>>(
```

Comment:
If I'm not mistaking array_slice is now in the corelib, so I think you can safely remove this :smile:
and update the relevant places